backend/src/SM.Service/Extensions/PatternExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace SM.Service
+{
+    public static class PatternExtensions
+    {
+        public static List<Resource> ToResourseList(this PatternItems patternItems)
```

Comment:
Same here, not quite sure that it makes sense to create such extension method. Usually extension methods make sense for some common cases. And this one will be used only in one place.